dev/Vagrantfile

```diff
@@ -58,7 +58,6 @@ def define_chef_server(config, attributes)
              "opscode_erchef['db_pool_size']" => 10,
              "oc_id['db_pool_size']" => 10,
              "oc_bifrost['db_pool_size']" => 10 }
-      # TODO merge with other options nicely.
       json = simple_deep_merge(json, { "provisioning" => { "chef-server-config" => pg } })
```

Comment:
nice spot.   The actual change was adding that `simple_deep_merge` and the to-do was a leftover for when that wasn't done